client.go

```diff
@@ -82,11 +83,18 @@ func (c *Client) do(method, path string, body interface{}, result interface{}) e
 	if !isHTTPSuccess(resp.StatusCode) {
 		defer resp.Body.Close()
 		body, _ := ioutil.ReadAll(resp.Body)
+
+		// Dump request headers sans Authorization.
+		var b bytes.Buffer
+		writer := bufio.NewWriter(&b)
+		_ = req.Header.WriteSubset(writer, map[string]bool{"Authorization": true})
```

Comment:
You could make this map once instead as a package var.

Reply:
oh good call